docs/dependencies.rst

```diff
@@ -92,3 +92,44 @@ need to use them (the most common base app is used for `Electron applications
 <https://github.com/flathub/io.atom.electron.BaseApp>`_). However, if your
 application uses a large, complex or specialized framework, it is a good
 idea to check for available base apps before you start building.
+
+Extensions
+----------
+
+Runtimes and applications can define extension points, which allow optional,
+additional runtimes to be mounted at a specified location inside the sandbox
+when they are present on the system. Typical uses for extension points include
+translations for applications, debuginfo for sdks, or adding more functionality
+to the application. These extensions are also referred to as "Add-ons".
```

Comment:
AFAIK, only GNOME-Software calls them Add-ons. Its not really much of a term in flatpak.

Reply:
I think referencing Add-ons would help if a developer ever searched for that term. What do you think?

Reply:
Maybe different working like `Some software refers to these extensions as "Add-ons".`